Vagrantfile

```diff
@@ -1,19 +1,21 @@
 # frozen_string_literal: true
 
-Vagrant.configure(2) do |config|
-  config.vm.box = 'remram/debian-9-i386'
+Vagrant.configure("2") do |config|
+  config.vm.box = "roboxes-x32/debian11"
 
   # Sometimes, the default seems to be too little for this one.
   config.vm.boot_timeout = 600
-
+  # When using libvirt it's usefull to explicitly tell Vagrant what method
+  # of folder syncing to use.
+  config.vm.synced_folder "./", "/vagrant", type: "rsync"
   config.vm.provision 'shell', inline: <<-SHELL
     set -e
 
     # Override sources.list for faster downloads (for people outside of the US).
-    echo deb http://httpredir.debian.org/debian/ stretch main > /etc/apt/sources.list
-    echo deb-src http://httpredir.debian.org/debian/ stretch main >> /etc/apt/sources.list
-    echo deb http://security.debian.org/debian-security stretch/updates main >> /etc/apt/sources.list
-    echo deb-src http://security.debian.org/debian-security stretch/updates main >> /etc/apt/sources.list
+    echo deb http://httpredir.debian.org/debian/ bullseye main > /etc/apt/sources.list
+    echo deb-src http://httpredir.debian.org/debian/ bullseye main >> /etc/apt/sources.list
+    # echo deb http://security.debian.org/debian-security bullseye/updates main >> /etc/apt/sources.list
+    # echo deb-src http://security.debian.org/debian-security bullseye/updates main >> /etc/apt/sources.list
```

Comment:
Any particular reason the security updates are commented out?

Reply:
No not really, I guess I just didn't think I needed them for just building this. I'll enable them again, probably a good thing to have.